<launch>
    <group ns='leo_locomotion'>
        <node name='map_design' pkg='leo_rover_localization' type='MapDesgin.py' args="$(find leo_rover_localization)/resources/leo_map.npz" />

        <node name="path_planner" pkg="leo_rover_localization" type="path_planner_service.py" args="$(find leo_rover_localization)/resources/leo_map.npz" output='screen' />

        <node name="rover_locomotion" pkg="leo_rover_localization" type="rover_locomotion_node.py" output='screen' />
        <rosparam command='load' file='$(find leo_rover_artag)/params/ar_tags.yaml' />

        <node name="rover_controller" pkg="leo_rover_localization" type="rover_controller_node.py" output="screen" />
        <rosparam command="load" file="$(find leo_rover_localization)/params/waypoints.yaml" />
    </group>
</launch>